packages/gatsby-link/package.json

```diff
@@ -12,6 +12,9 @@
     "@types/reach__router": "^1.0.0",
     "prop-types": "^15.6.1"
   },
+  "peerDependencies": {
```

Comment:
Want to move this over to the section below? 🙂 There's already `peerDependencies` set at the bottom! Also maybe setting it to map to the react version is a good idea too. So in this case, that would be "react-dom": "^16.3"

Reply:
Nice catch!

Should probably map to the same versions we use in gatsby proper, too.

https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/package.json#L161-L164

Reply:
Updated to `^16.4.2`! 👍